src/app/components/lineage-graph/lineage-graph.component.html

```diff
@@ -1,4 +1,4 @@
-<div #containerRef class="position-relative">
+<div #containerRef class="position-relative" *ngIf="graph.links && graph.nodes">
```

Comment:
Links and nodes are arrays, I think it's better to check their length
```suggestion
<div #containerRef class="position-relative" *ngIf="graph.links.length && graph.nodes.length">
```

Reply:
Ok